nix/default.nix

```diff
@@ -1,27 +1,51 @@
-{ elasticsearch-port ? 19200, nixpkgsPath, hspkgs, self }:
+{ elasticsearch-port ? 19200, nixpkgsPath, latestnixpkgsPath, self }:
 let
   nixpkgsSrc = import nixpkgsPath;
+  latestnixpkgsSrc = import latestnixpkgsPath;
 
   rev = if self ? rev then
-    self.rev
+  self.rev
   else
-    throw "Refusing to build from a dirty Git tree!";
+  throw "Refusing to build from a dirty Git tree!";
 
   src = pkgs.lib.cleanSourceWith {
     src = self; # The original, unfiltered source
     filter = path: type:
-      type == "directory" || (pkgs.lib.hasSuffix ".cabal" path)
-      || (pkgs.lib.hasSuffix ".hs" path) || (pkgs.lib.hasSuffix ".dhall" path)
-      || (pkgs.lib.hasSuffix ".json" path) || (pkgs.lib.hasSuffix ".yaml" path)
-      || (pkgs.lib.hasSuffix "LICENSE" path)
-      || (pkgs.lib.hasSuffix ".graphql" path);
+    type == "directory" || (pkgs.lib.hasSuffix ".cabal" path)
+    || (pkgs.lib.hasSuffix ".hs" path) || (pkgs.lib.hasSuffix ".dhall" path)
+    || (pkgs.lib.hasSuffix ".json" path) || (pkgs.lib.hasSuffix ".yaml" path)
+    || (pkgs.lib.hasSuffix "LICENSE" path)
+    || (pkgs.lib.hasSuffix ".graphql" path);
 
   };
 
   # Add monocle and patch broken dependency to the haskell package set
   haskellExtend = hpFinal: hpPrev: {
     monocle = hpPrev.callCabal2nix "monocle" src { };
 
+    # there is a test failure: resolveGroupController should resolve a direct mount root
+    cgroup-rts-threads = pkgs.haskell.lib.dontCheck
+    (pkgs.haskell.lib.overrideCabal hpPrev.cgroup-rts-threads {
+      broken = false;
+    });
+
+    # Gerrit needs HEAD
```

Comment:
that's silly, let's capture a task to make a new release!

Reply:
https://github.com/change-metrics/monocle/issues/1138